template/tsconfig/vitest/tsconfig.vitest.json

```diff
@@ -1,11 +1,18 @@
 {
   "extends": "./tsconfig.app.json",
```

Comment:
I kept this extending the app config, but maybe it's better to make it extend @vue/tsconfig/tsconfig.dom.json directly? That's how I've set it up in my project, makes it easier to manage.

Reply:
Yeah, I'd argue it's better to keep production and test configs separate (and rather have them extend a common base config), so that people won't e.g. add types to `tsconfig.app.json` that don't belong in test files.